bedrock/externalpages/templates/externalpages/pocket/get-inspired.html

```diff
@@ -0,0 +1,81 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+  {{ css_bundle('pocket-get-inspired') }}
+{% endblock  %}
+
+{% block page_title %}Get Inspired{% endblock %}
+
+{% block content %}
+
+{% include 'externalpages/pocket/includes/nav.html' %}
+
+<section>
+{% call split(
+  block_class='mzp-l-split-center-on-sm-md',
+  media_class='mzp-l-split-h-center',
+  image_url='img/externalpages/pocket/save-inspirations.png',
```

Comment:
```suggestion
  image_url='img/externalpages/pocket/save-inspirations.png',
  media_after=True
```
To get text first mobile stack

Reply:
On the [original site](https://getpocket.com/en/get-inspired/) the media comes before the text, though?

Reply:
this is what I see on the live site
<img width="413" alt="Screen Shot 2021-12-17 at 12 03 12 PM" src="https://user-images.githubusercontent.com/19650432/146542137-4fb010a1-bb79-4015-81e0-c8bffcb79ba2.png">

Reply:
Ah, this is not possible to recreate with split because the text and buttons are separated on mobile. I would still prefer text first and CTA buttons easier to reach.

Reply:
@reemhamz I'm approving as is. Split works well enough and media first avoids a clash on the image below. Sorry for the confusion.